src/main/java/com/github/wovnio/wovnjava/PathUrlLanguagePatternHandler.java

```diff
@@ -26,6 +30,10 @@ String insertLang(String url, String lang) {
         return "site.com/en/path";
     }
 
+    boolean isMatchSitePrefixPath(String url) {
```

Comment:
I think it should be named `isMatchingSitePrefixPath` or `matchesSitePrefixPath`.

Reply:
Now called `isMatchingSitePrefixPath`